Documentation/using-rkt-with-kubernetes.md

```diff
@@ -0,0 +1,19 @@
+# Using rkt with Kubernetes (aka "rktnetes")
+
+[Kubernetes](https://kubernetes.io) is a system for managing containerized applications across a cluster of machines.
+Kubernetes runs all applications in containers.
+In the default setup, this is performed using the Docker engine, but Kubernetes also features support for using rkt as its container runtime backend.
+This allows a Kubernetes cluster to leverage some of rkt's security features and native pod support.
+
+## Configuring rkt as the Kubernetes container runtime
+
+The container runtime is configured at the kubelet level.
+
+** TODO: add example here... **
```

Comment:
Kubelet (the agent that runs on each machine to manage containers) provides several flags to use rkt as the container runtime. Including:
- `--container-runtime=rkt` Choose rkt as the runtime.
- `--rkt-path` Set the rkt binary path.
- `--rkt-stage1-image` Set the stage1 image path.
 
For more detailed information about how to launch a kubernetes cluster with rkt, please view [this guide](
https://github.com/kubernetes/kubernetes/blob/master/docs/getting-started-guides/rkt/README.md)